CMakeLists.txt

```diff
@@ -377,6 +399,11 @@ if(UNIX)
   install(FILES LICENSE DESTINATION ${CMAKE_INSTALL_DOCDIR})
   install(FILES LICENSE ${CMAKE_CURRENT_BINARY_DIR}/libuv.pc
           DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
-  install(TARGETS uv LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
-  install(TARGETS uv_a ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+  if(CYGWIN)
+    install(TARGETS uv_shared RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+    install(TARGETS uv_shared ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+  else()
+    install(TARGETS uv_shared LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+  endif()
```

Comment:
Installation of `uv_shared` can be simplified to

```cmake
install(TARGETS uv_shared
  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
  )
```

CMake will automatically choose the proper subset of artifacts to install on each platform.